index.js

```diff
@@ -25,6 +25,9 @@ function mewt(target) {
       return mewt(newObj)
     },
     $unset: prop => {
+      if (isA && Number.isInteger(prop)) {
+        return mewt([...target.slice(0, prop), ...target.slice(prop + 1)])
```

Comment:
Yep... Not sure which one is the best. 
- `filter` is a for loop with a condition on each element and also filling up a new array one by one.
- `slice` (or what I did in the current impl) is probably the worst for big arrays since it slices it in half (2 relatively big arrays or 1 large 1 small) then immediately throws away both and creates a 3rd big one. 
- `clone` & `splice` is probably the best in terms of performance, since we only clone once then do the magic internally with a mutation - and it's probably fairly cheap and optimized.

But I think `filter` and `clone` & `splice` should be relatively close. However we never know for sure without benchmarks :)